public/scripts/client-helpers.js

```diff
@@ -224,3 +241,21 @@ const formatDate = (date) => {
 
   return `${month} ${dateArr[2]}, ${dateArr[0]}`;
 };
+
+const setDefaultDate = function() {
```

Comment:
Nice I kept thinking about this one when I had to click a date every time to test submission.